test/cli/__snapshots__/basic.test.js.snap.webpack4

```diff
@@ -80,6 +80,7 @@ Options:
   --client-web-socket-url-password <value>  Tells clients connected to devServer to use the provided password to authenticate.
   --web-socket-server <value>               Allows to set web socket server and options (by default 'ws').
   --no-web-socket-server                    Negative 'web-socket-server' option.
+  --web-socket-server-type <value>          Allows to set web socket server and options (by default 'ws').
```

Comment:
Can we mark `--web-socket-server <value>` is deprecated and add TODO?

Reply:
Yes, I will update.